src/WebJobs.Extensions.Twilio/Config/TwilioSMSConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Twilio
+{
+    /// <summary>
+    /// Defines the configuration options for the Twilio binding.
+    /// </summary>
+    public class TwilioSmsConfiguration
+    {
+        internal const string AzureWebJobsTwilioAccountSidKeyName = "AzureWebJobsTwilioAccountSid";
+        internal const string AzureWebJobsTwilioAccountAuthTokenName = "AzureWebJobsTwilioAuthToken";
+
+        public TwilioSmsConfiguration()
+        {
+            var nameResolver = new DefaultNameResolver();
+            AccountSid = nameResolver.Resolve(AzureWebJobsTwilioAccountSidKeyName);
```

Comment:
Was following the pattern used by the sample and Sendgrid. I'll update. Do we need to go through some of the other extensions and do the same?